### PR TITLE
Update problem definition to be more specific

### PR DESCRIPTION
**Pull request by Pattrigue**
According to supervisor feedback about lack of specificity.

### CODE REVIEW

File: report_pre_thesis/src/index.tex [hunk 1]
```diff
@@ -70,9 +72,32 @@ \subsection{Problem Statement Proposal}
...
+
+\begin{definition}
+    A LIBS data set $L$ for Martian geological samples is a subset of \(K' \times S \) where \(K' \subseteq K \), defined as \(L = \{(k_1, s_1 , (k_2 , s_2)) \dots (k_m , s_m) \} \). Each pair $(k_i , s_i)$ in $L$ represents the spectral data for an individual Martian geological sample, where $k_i$ is the sample's composition and $s_i$ is its emitted spectrum produced via laser-induced breakdown.
+\end{definition}
+
+\begin{definition}
+    A Calibration Set $C$ is a subset of \(K' \times S \) where \(K' \subseteq K \), defined as \(C = \{(k_1, s_1 , (k_2 , s_2)) \dots (k_n , s_n)\} \). Each pair $(k_i , s_i)$ in $C$ represents a known composition \(k_i \in K' \) from Earth's rock standards and its corresponding emitted spectrum \(s_i \in S \). Additionally, denoting the set of compositions associated with $L$ as $K_L$, then \( \forall(k,s) \in C, k \notin K_L \).
+\end{definition}
```
Comment (chhoumann) [inline report_pre_thesis/src/index.tex:84]: Well done on formulating this mathematically, @Ivikhostrup.

What's the reasoning behind the following?
> Additionally, denoting the set of compositions associated with $L$ as $K_L$, then \( \forall(k,s) \in C, k \notin K_L \).

I think, after this section, it would be interesting to write more about the actual predictions: how they're derived, from what, etc.
However, an issue with starting entirely from scratch, as we do here, is that we'd need to also describe the entire preprocessing stage in mathematical terms. And we'd need to do so before we can even get to the predictions.

We'll need to describe the exact data that the model takes in. Perhaps we can skip the preprocessing stage's mathematical description and just say:
$C$ and $L$ undergo various preprocessing steps to normalize and adjust the data such that the various noisy components have been removed. The output of this process is data in the cleaned, calibrated spectra format, which follows the same format as previously described, except the data has been preprocessed. The primary difference is the intensity measure is now in units of photons per channel in the spectral intensity data.

And then something like:
Given $(k_i , s_i)$ in $C$, a calibration model can be trained on $s_i$ to predict $k_i$.

Where you could say that the PLS-SM system uses multiple such models, each trained on a subset of each $s_i$ where the $s_i$ is partitioned into groups given its wavelengths. Then they become 'experts' at predicting compositions in that range.

But perhaps we lack some definition of $s_i$, because it would perhaps be a matrix of wavelengths associated with intensities? In which case the symbol isn't representative by convention.

Thoughts?

File: report_pre_thesis/src/index.tex [hunk 2]
```diff
@@ -73,7 +73,9 @@ \section{Related work}
 % \subsection{comment-end}
 
 \section{Definition}
-Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS. Additionally, let $S$ be the set of all possible spectra emitted from these compositions.
+Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS.
+
+Let $S$ be the set of all possible spectra, where each spectrum is a multidimensional vector in $\mathbb{R}^n$ where $n$ being the number of wavelengths captured by the LIBS spectrometer. For any spectrum $s_i$, it holds that \(s_i \in S \).
```
Comment (chhoumann) [inline report_pre_thesis/src/index.tex:78]:
```suggestion
Let $S$ be the set of all possible spectra, where each spectrum $\mathbf{s} \in S$ is a multidimensional vector in $\mathbb{R}^n$ with $n$ being the number of wavelengths captured by the LIBS spectrometer for a single laser pulse. $\mathbf{s}$ holds the intensities of light emitted from the plasma that is created from the laser ablation process.
```

Likely closer, no?

Comment (chhoumann) [reply report_pre_thesis/src/index.tex:78]: So you can see it rendered:

Let $S$ be the set of all possible spectra, where each spectrum $\mathbf{s}\_{i} \in S$ is a multidimensional vector in $\mathbb{R}^{n}$ with $n$ being the number of wavelengths captured by the LIBS spectrometer for a single laser pulse. $\mathbf{s}_{i}$ holds the intensities of light emitted from the plasma that is created from the laser ablation process.

Comment (chhoumann) [reply report_pre_thesis/src/index.tex:78]: Bolded $\mathbf{s}$ because conventionally vectors are lowercase bold.

File: report_pre_thesis/src/index.tex [hunk 3]
```diff
@@ -84,15 +86,7 @@ \section{Definition}
...
-    A prediction set $\mathbf{\hat{y}}$ is the weigthed average of the predictions from the PLS and ICA models defined by:
-
-    \( \mathbf{\hat{y}} = w_{PLS} \times Y_{PLS} + w_{ICA} \times Y_{ICA} \)
-
-    Let:
-    \begin{itemize}
-        \item $Y_{PLS}$ and $Y_{ICA}$ be the prediction sets from the PLS and ICA models, respectively.
-        \item $w_{PLS}$ and $w_{ICA}$ be the weights assigned to the PLS and ICA models, respectively, such that \(w_{PLS} + w_{ICA} = 1 \), ensuring a weighted average.
-    \end{itemize}
+    Let \(h: S \rightarrow \mathbb{R}^m \) be a hypothesis function, where for each spectrum \(s \in S\), $h(s)$ produces a prediction \(\mathbf{\hat{y}} \in \mathbb{R}^m \). Here, $m$ denotes the dimensions of the composition predictions, and the structure and content of $\mathbf{\hat{y}}$ are analogous to a composition $k$. The function $h$ is determined by the model's learned parameters and the hyperparameters of the algorithm.
```
Comment (chhoumann) [inline report_pre_thesis/src/index.tex]: - Remember that the (current) models do not take raw spectra.
- We do not define the units, merely that they are real numbers
- "_Here, $m$ denotes the dimensions of the composition predictions_" seems unclear
- Don't think we can just say 'seems analogous'
- My remark on "_The function $h$ is determined by the model's learned parameters and the hyperparameters of the algorithm._" was just for clarification between us - it's hopefully implicitly understood by anyone reading, as it is common knowledge in ML research.

File: report_pre_thesis/src/index.tex [hunk 4]
```diff
@@ -59,7 +59,9 @@ \section{Introduction}
 
 Consequently, there is a need to not only identify and examine the existing model's limitations but also to understand precisely why improvements would be beneficial and how they can be effectively implemented.
 
-\subsection{Problem Statement Proposal}
+As such, we aim to identify potential limitations and areas of improvement in the Multivariate Oxide Composition model's predictive accuracy and robustness in predicting the composition of major oxides in Martian geological samples using Earth-based calibration data.
+
+\section{Related work}
```
[the remark this answers is not in the document]
Comment (Ivikhostrup) [reply report_pre_thesis/src/index.tex:64]: I would be inclined to agree. This suggestion was based on what I have seen other papers do, but I do not have a strong opinion on the matter.

File: report_pre_thesis/src/index.tex [hunk 5]
```diff
@@ -73,7 +73,9 @@ \section{Related work}
 % \subsection{comment-end}
 
 \section{Definition}
-Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS. Additionally, let $S$ be the set of all possible spectra emitted from these compositions.
+Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS.
```
Comment (chhoumann) [inline report_pre_thesis/src/index.tex:76]: I think this can be made clearer.

$E$ has, as of our current understanding, at most 118 elements (how many there are in the periodic table). Then $\mathbb{P}(E)$ would be of size $2^{118}=3.32307e+35$.

And then we limit our consideration to $K$, a subset of $\mathbb{P}(E)$, to some number that I personally don't know - I suspect none of us do.

In reality, the interest of the ChemCam MOC model is only predicting the compositions of 8 major oxides.
Could we simply give those and state that we intend to only look at those?

In other words:
Does the implication of us including (unknown, to us) other elements in our definitions benefit our analysis?
I don't know whether it could, but I think we may benefit from simply sticking to what the experts do (in the papers we use as baseline).

Also, just as an aside: isn't it convention to denote the power-set by $\mathcal{P}(E)$? In my experience, $\mathbb{P}$ is primarily used to denote number fields, e.g. $\mathbb{R}$.

**Review by chhoumann — COMMENTED** [review]
Some surface-level comments on the current iteration. Sorry for the lack of in-depth review - don't have much time right now :)

File: report_pre_thesis/src/index.tex [hunk 6]
```diff
@@ -70,9 +72,26 @@ \subsection{Problem Statement Proposal}
...
 % \subsection{comment-end}
 
-In this work, we aim to identify potential limitations and areas of improvement in the Multivariate Oxide Composition model's predictive accuracy and robustness in predicting the composition of major oxides in Martian geological samples using Earth-based calibration data.
+\section{Definition}
+Given a set \(E\) of eight major oxides detectable using LIBS, \(E = \{SiO_2, TiO_2, Al_2 O_3, FeOT, MgO, CaO, Na_2O, K_2O\}\), and considering \(\mathcal{P}(E)\) as the power set of \(E\) which represents all possible combinations of these oxides, we define \(K \subseteq \mathcal{P}(E) \) as the set of oxide compositions detected by LIBS.
+
+Let $S$ be the set of all possible spectra, where each spectrum $\mathbf{s} \in S$ is a multidimensional vector in $\mathbb{R}^n$ with $n$ being the number of wavelengths captured by the LIBS spectrometer for a single laser pulse. $\mathbf{s}$ holds the intensities of light emitted from the plasma that is created from the laser ablation process.
```
Comment (chhoumann) [inline report_pre_thesis/src/index.tex:78]: Actually, we can't do this because we need to distinguish wavelengths! The value of the wavelength matters greatly in the predictions, because you need to consider the wavelength where you see a high/low intensity

**Comment by chhoumann** [comment]
Redundant by #10
